<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>充电计费用户客户端</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="account" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>401</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>账号管理</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="l4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>身份</string>
    </property>
   </widget>
   <widget class="QLabel" name="l5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userState">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="userIdentity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="charge" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>充电管理</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>充电模式</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chargeMode">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>快充</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>正常</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>电池容量（新请求）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请求电量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="batteryCapacity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="electricityReq">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chargeReqBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提交充电请求</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rewriteBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endChargeBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消/结束充电</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="info" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>581</width>
     <height>831</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>信息查询</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询详单</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="infoTable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>详单编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>生成时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>充电桩编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>充电电量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>充电时长</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>启动时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>停止时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>充电费用</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>服务费用</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>总费用</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>129</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>当前状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>690</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>排队号码</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="presentState">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="presentQueue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>630</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="queueNum">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>690</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
